epix/clustering.py

```diff
@@ -28,6 +28,11 @@ def find_cluster(interactions, cluster_size_space, cluster_size_time):
 
     # Splitting into individual events and apply time clustering:
     groups = df.groupby('entry')
+    if len(groups.count()) == 0:
+        # TPC interaction is empty
+        #interactions = []
+        return  interactions
```

Comment:
I think you can already check the length of df before the grouping is not it? The groups can be only all empty if df is already empty.

Reply:
fixed, Thank you!